lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
@rlucioni here it is making a group of all js files.
https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1279